internal/exec/vendor_utils.go

```diff
@@ -417,7 +418,7 @@ func ExecuteAtmosVendorInternal(
 	// Run TUI to process packages
 	if len(packages) > 0 {
 		var opts []tea.ProgramOption
-		if !CheckTTYSupport() {
+		if !term.CheckTTYSupportForStdout() {
```

Comment:
Whatever we do here, we should do it consistently. We have so many of these check tty functions and implementations.

Reply:
Done